configs/SKYSTARSF435PRO/config.h

```diff
@@ -0,0 +1,120 @@
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU        AT32F435M
+
+#define BOARD_NAME           SKYSTARSF435PRO
+#define MANUFACTURER_ID      SKST
+
+#define USE_ACC
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASH_M25P16
+#define USE_MAX7456
+
+#define BEEPER_PIN           PB2
+#define MOTOR1_PIN           PB0
+#define MOTOR2_PIN           PB1
+#define MOTOR3_PIN           PC8
+#define MOTOR4_PIN           PC9
+#define MOTOR5_PIN           PB6
+#define MOTOR6_PIN           PB7
+#define SERVO1_PIN           PB10
+#define SERVO2_PIN           PB11
+//#define RX_PPM_PIN           PA8
+#define LED_STRIP_PIN        PA8
+#define UART1_TX_PIN         PA9
+#define UART2_TX_PIN         PA2
+#define UART3_TX_PIN         PC10
+#define UART4_TX_PIN         PA0
+#define UART5_TX_PIN         PC12
+#define UART6_TX_PIN         PC6
+#define UART1_RX_PIN         PA10
+#define UART2_RX_PIN         PA3
+#define UART3_RX_PIN         PC11
+#define UART4_RX_PIN         PA1
+#define UART5_RX_PIN         PD2
+#define UART6_RX_PIN         PC7
+#define I2C1_SCL_PIN         PB8
+#define I2C1_SDA_PIN         PB9
+#define I2C2_SCL_PIN         PH2
+#define I2C2_SDA_PIN         PH3
+#define LED0_PIN             PC3
+#define SPI1_SCK_PIN         PA5
+#define SPI2_SCK_PIN         PB13
+#define SPI3_SCK_PIN         PB3
+#define SPI1_SDI_PIN         PA6
+#define SPI2_SDI_PIN         PB14
+#define SPI3_SDI_PIN         PB4
+#define SPI1_SDO_PIN         PA7
+#define SPI2_SDO_PIN         PB15
+#define SPI3_SDO_PIN         PB5
+//#define CAMERA_CONTROL_PIN   PC9
+#define ADC_RSSI_PIN         PC2
+#define ADC_CURR_PIN         PC1
+#define PINIO1_PIN           PC0
+#define PINIO2_PIN           PC5
+#define FLASH_CS_PIN         PA15
+#define MAX7456_SPI_CS_PIN   PB12
+#define GYRO_1_EXTI_PIN      PC4
+#define GYRO_1_CS_PIN        PA4
+
+#define TIMER_PIN_MAPPING    TIMER_PIN_MAP( 0, PB0,  2,  1 ) \
+                             TIMER_PIN_MAP( 1, PB1,  2,  2 ) \
+                             TIMER_PIN_MAP( 2, PC8,  2,  3 ) \
+                             TIMER_PIN_MAP( 3, PC9,  2,  4 ) \
+                             TIMER_PIN_MAP( 4, PB6,  1,  5 ) \
+                             TIMER_PIN_MAP( 5, PB7,  1,  6 ) \
+                             TIMER_PIN_MAP( 6, PB10, 1, -1 ) \
+                             TIMER_PIN_MAP( 7, PB11, 1, -1 ) \
+                             TIMER_PIN_MAP( 8, PA8,  1,  7 )
+
+
+#define ADC_INSTANCE                 ADC1
```

Comment:
ADC1 is default instance
```suggestion
#define ADC1_DMA_OPT                 8
```

Reply:
Good catch!